docs/containers-policy.json.5.md

```diff
@@ -90,7 +90,9 @@ Scopes matching individual images are named Docker references *in the fully expa
 using a tag or digest. For example, `docker.io/library/busybox:latest` (*not* `busybox:latest`).
 
 More general scopes are prefixes of individual-image scopes, and specify a repository (by omitting the tag or digest),
-a repository namespace, or a registry host (by only specifying the host name).
+a repository namespace, or a registry host (by only specifying the host name)
+or a wildcarded expression for matching all subdomains. For wildcarded subdomain
+matching, `*.example.com` is a valid case, but `example*.*.com` is not.
```

Comment:
`atomic:` above (c/image/openshift) also needs updating. (Possibly also `docker-daemon:`, but that’s currently not documented here at all, so N/A.)

Reply:
Ack, so I could just copy-paste that paragraph on general scopes as-is to the atomic: section, correct? Or, are there any atomic specific changes required there?

Reply:
Yes; the two differ slightly in the after-hostname part (and the mandatory `//` prefix) only.

Reply:
pushed a new change, PTAL. Thanks

Reply:
ah, noticed your comment only now. I need to change this, sorry i just copy-pasted it earlier.

Reply:
@mtrmac re-read your comment just now. So, if the difference is only in the prefix and after-hostname, we should be good to go afaict. Let me know, thanks!